themes/admin/Sea_Green/css/sea_green.css

```
@@ -16342,7 +16381,7 @@ input[type=checkbox]:checked:after,
 
 #pjax-file-load-container > div {
   background: black;
-  background: linear-gradient(to right, black 0%, #081709 50%, #163c19 100%);
+  background: linear-gradient(to right, black 0%, rgb(8.4239130435, 22.5760869565, 9.2663043478) 50%, rgb(22.2826086957, 59.7173913043, 24.5108695652) 100%);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)